### ISSUE TITLE
Add a description fields in the AdminUI

### ISSUE THREAD

**Issue by jesstelford** [keystonejs/keystone#731]
Updated based on comments below...

`schemaDoc` is for developer docs and `adminDoc` is for AdminUI.

The action here is to allow a `adminDoc` key on lists:

```
fields: {
  name: {
    type: Text,
    schemaDoc: "This will show up in GraphiQL",
    adminDoc: "This will show up in the Admin UI",
  }
}
```

**Comment by MadeByMike** [comment]
Can we do this in the AdminUI in a standard way without requiring every field implementation to be updated?

**Comment by gautamsi** [comment]
I think admin ui should not be making description on its own.

Basic implementation should be quick one. 
Should this be a "good first issue"

**Comment by MadeByMike** [comment]
@gautamsi this just means that the admin UI would look for a description and display help text in a consistent way rather than having this be a field concern. The adminUI already makes thousands of decisions like this in how it displays any information about a list or field... this is far less opinionated than most of them.

**Comment by timleslie** [comment]
Fixed in #2287
